### PR TITLE
chore(design): improve design and compatibility

### PR DESCRIPTION
**Pull request by Thomaash**
There were two attempts at this already but it seems both gave up so I did it myself.

# IE11
It still doesn't work perfectly but the page is at least usable.

Issues:
- CircleCI logo is stretched (minor cosmetic issue, I don't think it's worth the hassle).
- Open Collective doesn't work (it's injected by their script, I have no idea how to fix this).

# All Browsers
Changes:
- All the things on the page are centered all the time.
- The article dividing lines stretch across the whole screen instead of mysteriously ending mid air.
- Open Collective no longer stretches the page after the window is shrunk.

# Code
I tried not to mess with the code style of CSS as I mostly deleted and added new stuff. I originally planned to do the same with HTML but I had to change the structure which resulted in 95 % of lines being changed and diff getting hopelessly out of sync. So I ran the entire thing through Prettier to at least get neat formatting.

I changed many weird things like `<ul><li>` being restyled in such a way to behave as `<div><span>` (any idea why this was done?) and also reduced duplicities, removed unused classes, styles etc.

# Screenshots
## Wide Before
screencapture-visjs-org-2019-11-27-21_53_06

## Wide After
screencapture-192-168-1-106-8000-2019-11-27-21_53_11

## Narrow Before
screencapture-visjs-org-2019-11-27-21_53_34

## Narrow After
screencapture-192-168-1-106-8000-2019-11-27-21_53_41

# Closes
- Closes #11.
- Closes #14.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
+
 header {
 	padding: 1em;
-	text-align: center;
 
 	color: white;
 	text-shadow: 1px 1px 3px rgba(0, 0, 0, 1);
 
-	background: var(--vis-color-blue)9; /* Old browsers */
+	background: #07508E; /* Old browsers */
```
Comment (suraj-sloth) [inline index.css]: Pardon me, but I wonder why did you remove the CSS variables part. Using CSS Variables makes the color scheme of website more flexible.

Comment (Thomaash) [reply index.css]: And absent in IE 11.
